nim/happyx/config.yaml

```diff
@@ -2,5 +2,5 @@ framework:
   github: HapticX/happyx
   version: 3.8
 
-build_opts: "-d:beast -d:disableApiDoc -d:useMalloc --threads:on --opt:speed"
+build_opts: "-d:beast -d:disableApiDoc -d:disableDefDeco -d:useMalloc --threads:on --opt:speed"
```

Comment:
do you recommand this for production usage ?

Reply:
yeap, this argument removes default decorators like `AuthBasic` and `GetUserAgent`